src/plugin/file/kml/kmlnodelayerui.js

```diff
@@ -1092,6 +1092,46 @@ plugin.file.kml.KMLNodeLayerUICtrl.prototype.createFeatureCommand = function(com
 };
 
 
+/**
+ * Creates a sequence of commands to run on each feature
+ *
+ * @param {function(string, string):os.command.ICommand} commandFunction
+ * @param {string=} title
+ */
+plugin.file.kml.KMLNodeLayerUICtrl.prototype.createFeatureSequenceCommand = function(commandFunction, title) {
+  var cmds = [];
+
+  var items = /** @type {Array<!plugin.file.kml.ui.KMLNode>} */ (this.scope['items']);
+  if (items) {
+    for (var i = 0; i < items.length; i++) {
+      var feature = items[i].getFeature();
+      var source = items[i].getSource();
+      if (feature && source) {
+        var featureId = feature.getId();
+        var layerId = source.getId();
+        if (featureId && layerId) {
+          if (typeof featureId == 'number') {
```

Comment:
A simple way to do this is `featureId = String(featureId);`, and skip the type check. If it's already a string, it won't be changed.

The prior condition (`if (featureId ...)`) will also fail if the id is the number `0`, so `featureId != null` would be a better test. That won't catch an empty string but... I'm hoping we would never do that.